src/main/antlr4/org/hisp/dhis/parser/expression/antlr/Expression.g4

```diff
@@ -125,11 +125,11 @@ expr
 
     //  Data items (alphabetical)
 
-    |   it='#{' uid0=UID (wild1='.*')? '}'
-    |   it='#{' uid0=UID '.' uid1=UID '}'
-    |   it='#{' uid0=UID '.' uid1=UID wild2='.*' '}'
-    |   it='#{' uid0=UID '.*.' uid2=UID '}'
-    |   it='#{' uid0=UID '.' uid1=UID '.' uid2=UID '}'
+    |   it='#{' uid0=(UID | TAGGED_UID0) (wild1='.*')? '}'
+    |   it='#{' uid0=(UID | TAGGED_UID0) '.' uid1=(UID | TAGGED_UID1) '}'
+    |   it='#{' uid0=(UID | TAGGED_UID0) '.' uid1=(UID | TAGGED_UID1) wild2='.*' '}'
+    |   it='#{' uid0=(UID | TAGGED_UID0) '.*.' uid2=(UID | TAGGED_UID1) '}'
+    |   it='#{' uid0=(UID | TAGGED_UID0) '.' uid1=(UID | TAGGED_UID1) '.' uid2=(UID | TAGGED_UID2) '}'
```

Comment:
uid2 should not be tagged (either with TAGGED_UID1 or TAGGED_UID2). Just UID, since we are not making a resolver for it in the main code.

Reply:
Done